### DISCUSSION TITLE
something strange..

### DISCUSSION THREAD

**Discussion by magic-thomas** [Power-Components/livewire-powergrid#266] [General]
Something strange happens.

screen captures attached.

<img width="793" alt="스크린샷 2022-02-08 04 23 39" src="https://user-images.githubusercontent.com/69987486/152857459-fac19d91-9748-4166-a411-9175e182e19f.png">

<img width="640" alt="스크린샷 2022-02-08 04 23 29" src="https://user-images.githubusercontent.com/69987486/152857514-d2e70a75-cb45-4357-bdb7-e4d2015318cc.png">

How can I fix this ? 

I totally don't have idea what it is.

**Comment by JF-626164** [comment]
Interesting, have you tried swapping which theme you're using to see if goes away?
